Riot/Modules/Room/VoiceMessages/VoiceMessageMediaServiceProvider.swift

```diff
@@ -256,9 +262,14 @@ import MediaPlayer
             return
         }
         
-        let nowPlayingInfoCenter = MPNowPlayingInfoCenter.default()
-        nowPlayingInfoCenter.nowPlayingInfo = [MPMediaItemPropertyTitle: VectorL10n.voiceMessageLockScreenPlaceholder,
-                                               MPMediaItemPropertyPlaybackDuration: audioPlayer.duration as Any,
-                                               MPNowPlayingInfoPropertyElapsedPlaybackTime: audioPlayer.currentTime as Any]
+        // If we have a NowPlayingInfoProvider for this player
+        if let nowPlayingInfoProvider = nowPlayingInfoProviders.object(forKey: audioPlayer) {
+            nowPlayingInfoProvider.updatePlayingInfoCenter(forPlayer: audioPlayer)
+        } else {
+            let nowPlayingInfoCenter = MPNowPlayingInfoCenter.default()
+            nowPlayingInfoCenter.nowPlayingInfo = [MPMediaItemPropertyTitle: VectorL10n.voiceMessageLockScreenPlaceholder,
+                                        MPMediaItemPropertyPlaybackDuration: audioPlayer.duration as Any,
+                                MPNowPlayingInfoPropertyElapsedPlaybackTime: audioPlayer.currentTime as Any]
+        }
```

Comment:
If `PlayingInfoProviders` is the way we want to go we should move this bit to the the voice message side.